packages/xstate-graph/src/graph.ts

```diff
@@ -93,9 +94,7 @@ export function createDefaultMachineOptions<TMachine extends AnyStateMachine>(
         })
       ) as any[];
     },
-    fromState: machine.getInitialState(
-      undefined as any // TODO: figure out the simulation API
-    ) as StateFrom<TMachine>,
+    fromState: simulate(machine).getInitialState(),
```

Comment:
I this this is somewhat wrong because it creates a fresh (and disposable) simulation object. 

Consider those lines:
https://github.com/statelyai/xstate/blob/ede42d88c521a5c8d1acd4ccdbf4032eefb2f023/packages/xstate-graph/src/pathFromEvents.ts#L40-L45

Both `fromState = ` and `getAdjacencyMap` might create the simulation object and thus 2 different dummy actor contexts. If things would, anyhow, rely on the (widely understood) identity of the actor context then things could break in some subtle ways because the same actor context/simulation object is not reused consistently throughout `getPathsFromEvents` (the function containing the quoted lines of code).

I think that a good part of the reason behind introducing `simulate` is the fact that we can create the actor context on behalf of the user and pass it around correctly from the very start to the very end of the "simulation".